avatica-shaded/pom.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<!-- Licensed to the Apache Software Foundation (ASF) under one or more contributor
+license agreements. See the NOTICE file distributed with this work for additional
+information regarding copyright ownership. The ASF licenses this file to
+You under the Apache License, Version 2.0 (the "License"); you may not use
+this file except in compliance with the License. You may obtain a copy of
+the License at http://www.apache.org/licenses/LICENSE-2.0 Unless required
+by applicable law or agreed to in writing, software distributed under the
+License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+OF ANY KIND, either express or implied. See the License for the specific
+language governing permissions and limitations under the License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.confluent</groupId>
+        <artifactId>kafka-connect-storage-common-parent</artifactId>
+        <version>10.0.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>kafka-connect-storage-common-avatica-shaded</artifactId>
+    <name>kafka-connect-storage-common-avatica-shaded</name>
+    
+    <description>avatica shaded package to replace jackson dependencies without CVEs</description>
+    <inceptionYear>2020</inceptionYear>
+
+    <properties>
+        <avatica.version>1.8.0</avatica.version>
+        <jackson.version>2.10.5</jackson.version>
```

Comment:
What Jackson version is shaded in the original library? And how will we gain confidence that there are no compatibility problems by replacing that version with 2.10.5?

Should this use a variable defined in the parent POM? I guess maybe not, since it doesn't have to be the same Jackson version used by the connector since this is shaded and used only by the avatica library.

Reply:
I was thinking that we won't like to upgrade the shaded jackson as frequently, so pin here made sense. This is a recent clear version that seems to work. Trying earlier ones would probably bring in other CVEs.